src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -4,6 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
+    <VersionPrefix>11.0.0-preview</VersionPrefix>
```

Comment:
@dihan0604 I just now noticed, why version is mentioned twice and that also separate version number?
Also as this is changing the public surface area, your major version should increase.